tests/ffi/deny_set_scheduler/test.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -euvx
+
+. ../common/prepare.sh
+
+disk_cleanup
+prepare_test
+reload_config
+
+podman exec -it qm /bin/bash -c \
+         'podman run -d  --replace --name ffi-qm \
+          quay.io/centos-sig-automotive/ffi-tools:latest \
+          tail -f /dev/null'
+
+return_from_setscheduler=$(podman exec -it qm /bin/bash -c \
+         'podman exec -it ffi-qm ./QM/test_sched_setscheduler')
+
+
+if [[ "${return_from_setscheduler}" =~ "Failed to set scheduler: Operation not permitted" ]]; then
```

Comment:
How about we short a bit:
```
if [[ ! "${return_from_setscheduler}" =~ "Failed to set scheduler: Operation not permitted" ]];
then
    error_message "set_scheduler() syscall can be executed in QM."
    exit 1
fi
```

Reply:
(need to be tested)

Reply:
sure, updated it

Reply:
Just to confirm, we cannot use a single if here? Do we need the else statement ? (Just trying to understand the context).